www/index.html

```diff
@@ -359,6 +359,11 @@
   <script type="text/javascript" src="lib/framework7/js/framework7.min.js"></script>
   <script type="text/javascript" src="js/init-styles.js"></script>
   <script type="text/javascript" src="js/my-app.js"></script>
+  <script type="text/javascript">
```

Comment:
Still, why not put this right in js/my-app.js ?
I am resistant to all (handwritten) forms of js code in html files.